### DISCUSSION TITLE
How to make  "Use Separate Columns for Curves" unchecked CSV file by Pyaedt?

### DISCUSSION THREAD

**Discussion by hsgo2430** [ansys/pyaedt#4966] [Q&A] [answered]
Hi,

I have one question that make csv file.
I make report and export it to csv by using this code.
```
input1 = "dB(S(1,1)); dB(S(2,1)); dB(S(1,2)); dB(S(2,2))"

expression1 = input1.split(';')

hfss.post.create_report(expression1, plot_name="plot11")

hfss.post.export_report_to_csv(
    project_dir=r'C:\Users\hsgo2\tmp'
    plot_name="plot11"
)
```
As a result, the csv file came out like this picture
![image](https://github.com/user-attachments/assets/558bce95-f314-4414-9cc5-9ffcac8623db)

But I want the csv format like the picture below.
![image](https://github.com/user-attachments/assets/2c9789eb-c3ad-4bd9-93bf-621cf906ccfd)

In Ansys program, It can be made unchecking "Use Separate Columns for Curves" option,
but `export_report_to_csv` doesn't have "Use Separate Columns for Curves" Parameter.

How can I get csv file like Second picture using Pyaedt?

**Comment by Samuelopez-ansys** [comment, marked as answer]
@hsgo2430 This option was not implemented, I have just created a pull request, and I added it:
https://github.com/ansys/pyaedt/pull/4968

**Reply by hsgo2430** [reply]
Thank you so much Samuel!
